### PR TITLE
Remove Jasmine from sbtPluginTest.

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

**Comment by nicolasstucki** [comment]
Review by @sjrd

File: ci/matrix.xml [hunk 1]
```diff
@@ -238,7 +238,7 @@
     sbt ++2.10.6 ir/publishLocal tools/publishLocal jsEnvs/publishLocal \
```
Comment (sjrd) [inline ci/matrix.xml]: I guess you can also remove `jasmineTestFramework/publishLocal` 2 lines above this one, then.

File: sbt-plugin-test/multiTest/js/src/test/scala/sbttest/multitest/ScalaJSJUnitBootstrapTest.scala [hunk 2]
```diff
@@ -1,22 +1,20 @@
 package sbttest.multitest
 
+import scalajs.js
```
Comment (sjrd) [inline sbt-plugin-test/multiTest/js/src/test/scala/sbttest/multitest/ScalaJSJUnitBootstrapTest.scala]: `import scala.scalajs.js`

**Comment by sjrd** [comment]
That's all.

File: ci/matrix.xml (hunk 1 above)
Comment (nicolasstucki) [inline ci/matrix.xml]: Done

**Comment by sjrd** [comment]
LGTM

**Pull request merged**